package.json

```diff
@@ -67,7 +67,7 @@
     "typescript": "*"
   },
   "devDependencies": {
-    "@lifeomic/alpha": "^5.0.1",
+    "@lifeomic/alpha": "^4.1.0",
```

Comment:
Alpha 5.0 requires the AWS SDK 3.0 and this project is built around the 2.0 SDK. Don't pull the 3.0 SDK for development